chapters/ch06.md

```diff
@@ -1,45 +1,54 @@
 
 ->
+Tập kiểm tra khó hơn tập phát triển. Thuật toán của bạn lúc này đã làm tốt nhất và không thể cải thiện thêm, hoặc cải thiện không đáng kể.  
 
 3. The test set is not necessarily harder, but just different, from the dev set. So what works well on the dev set just does not work well on the test set. In this case, a lot of your work to improve dev set performance might be wasted effort.
 
--> 
+->
+Tập kiểm tra có thể không khó hơn nhưng lại khác so với tập phát triển. Do đó mô hình có thể thực thi tốt trên tập phát triển nhưng lại kém trên tập kiểm tra. Trong trường hợp này việc cố gắng cải thiện tập phát triển có thể trở nên vô nghĩa.
 
 Working on machine learning applications is hard enough. Having mismatched dev and test sets introduces additional uncertainty about whether improving on the dev set distribution also improves test set performance. Having mismatched dev and test sets makes it harder to figure out what is and isn’t working, and thus makes it harder to prioritize what to work on.
 
 ->
+Nhiều vấn đề phức tạp có thể xảy ra khi làm việc với các ứng dụng machine learning. Tập phát triển và tập kiểm tra khác nhau khiến chúng ta không chắc chắn về việc nên cải thiện tập phát triển hay cải thiện tập kiểm tra cùng lúc. Tập phát triển và tập kiểm tra khác nhau cũng khiến cho việc chuẩn đoán vấn đề trở nên phức tạp hơn, nhất là trong việc lựa chọn các công việc ưu tiên cần thiết để cải thiện mô hình.
 
 If you are working on a 3rd party benchmark problem, their creator might have specified dev and test sets that come from different distributions. Luck, rather than skill, will have a greater impact on your performance on such benchmarks compared to if the dev and test sets come from the same distribution. It is an important research problem to develop learning algorithms that are trained on one distribution and generalize well to another. But if your goal is to make progress on a specific machine learning application rather than make research progress, I recommend trying to choose dev and test sets that are drawn from the same distribution. This will make your team more efficient.
 
-->
\ No newline at end of file
+-> Nếu bạn đang làm việc cho một bên thứ 3 về vấn đề đánh giá xếp hạng, có khả năng mô hình của họ chứa các tập phát triển và tập kiểm tra có phân phối khác nhau. Điều may mắn là thay vì dựa trên kỹ năng, nó sẽ tác động đáng kể đến năng lực của mô hình trong cùng đánh giá xếp hạng so với trường hợp khi mà các tập phát triển và tập kiểm tra cùng đến từ một phân phối. Phát triển các thuật toán machine learning được huấn luyện trên một phân phối và thực thi tốt trên các phân phối khác là vấn đề quan trọng trong nghiên cứu. Tuy nhiên nếu mục tiêu của bạn là cải tiến một ứng dụng machine learning cụ thể nào đó thay vì mục đích làm nghiên cứu, tôi khuyên bạn nên chọn các tập phát triển và tập kiểm tra có chung phân phối. Điều này sẽ giúp cả nhóm làm việc hiệu quả hơn.
```

Comment:
Đoạn này khá khó dịch. Mình xin dịch lại toàn bộ, có thể như thế dễ hơn với mình:

> Nếu bạn đang làm việc với một bài toán đánh giá xếp hạng của bên thứ ba, họ có thể cung cấp tập phát triển và tập kiểm tra có phân phối khác nhau. Nếu so sánh với bài toán có tập phát triển và tập kiểm tra đến từ cùng phân phối, bài toán này có kết quả phụ thuộc nhiều vào may mắn hơn là kỹ năng của bạn. Việc phát triển các thuật toán mà mô hình được huấn luyện trên một phân phối để khái quát hóa tốt trên một phân phối khác là một vấn đề quan trọng trong nghiên cứu. Tuy nhiên, nếu mục tiêu của bạn là cải thiện một ứng dụng machine learning cụ thể thay vì tạo ra tiến triển trong nghiên cứu, tôi khuyên bạn chọn tập phát triển và tập kiểm tra đến từ cùng phân phối. Điều này sẽ khiến nhóm bạn làm việc hiệu quả hơn.